py4DSTEM/process/diskdetection/diskdetection_cuda.py

```diff
@@ -120,7 +121,7 @@ def find_Bragg_disks_CUDA(
     if get_maximal_points.max_threads_per_block < DP.shape[1]:
         # naive blocks/threads will not work, figure out an OK distribution
         blocks = ((np.prod(DP.shape) // get_maximal_points.max_threads_per_block + 1),)
-        threads = get_maximal_points.max_threads_per_block
+        threads = (get_maximal_points.max_threads_per_block,)
```

Comment:
curious why the tuple instead of an int?

Reply:
`cupy` requires this to be passed as a tuple when invoking a kernel. It is possible to set up higher dimension grids of threads, in which case this tuple has 2 or 3 items, but for launches that require multiple thread blocks it ends up being cleaner to do this using 1D thread groups.